### PR TITLE
Adds new integration [xilense/aimp_custom_component]

### PR DESCRIPTION
**Pull request by xilense**
Adds new integration [xilense/aimp_custom_component](https://github.com/xilense/aimp_custom_component)

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
YAML example given in README is missing the domain for the custom platform

**Review by ludeeus — CHANGES_REQUESTED** [review]
CI does not pass
https://hacs.xyz/docs/publish/include/#before-submitting

**Comment by xilense** [comment]
I'm sorry about issue, but I'm kind of new to github

when I'm running action hacs check it says

```
  ::INFO:: <Integration xilense/aimp_custom_component> hacs.json is valid
  Error:  ::error:: No file found 'manifest.json'
  Error: HACS load check
```

but im pretty sure `manifes.json` already in repo, 
what I should have to do?

**Comment by ludeeus** [comment]
<https://github.com/xilense/aimp_custom_component/blob/master/hacs.json#L3> this is worng

**Comment by xilense** [comment]
when I set `"content_in_root": true,` it says

```
  ::INFO:: <Integration xilense/aimp_custom_component> hacs.json is valid
  Error:  ::error:: No file found 'manifest.json'
  Error: HACS load check
```
but when i set `"content_in_root": false,` it says

```
  ::INFO:: <Integration xilense/aimp_custom_component> hacs.json is valid
  Error:  Repostitory structure for master is not compliant
  Error: HACS load check
```

i have updated my repo with no `INTEGRATION_NAME` directory, 
it _**success hacs check**_, but _**fail on hassfest check**_ with error:

```
Integration  - .:
Error: R] [MANIFEST] Domain does not match dir name
```

**Comment by ludeeus** [comment]
Right. Look at the structure examples in the docs

**Comment by xilense** [comment]
thanks for pointing my mistakes,

already set `hacs.json` manifest `"content_in_root": true,` but when I try structure examples:

### [Not OK example (1)](https://hacs.xyz/docs/publish/integration/#not-ok-example-1)
```
awesome/__init_.py
awesome/sensor.py
awesome/manifest.json
info.md
README.md
```
it says success on hassfest **fail on hacs** check with error:
```
  ::INFO:: <Integration xilense/aimp_custom_component> hacs.json is valid
  Error:  ::error:: No file found 'manifest.json'
  Error: HACS load check
```

### [Not OK example (2)](https://hacs.xyz/docs/publish/integration/#not-ok-example-2)
```
__init_.py
sensor.py
manifest.json
info.md
README.md
```
it says success on hacs check but **fail on hassfest** check with error
```
Integration  - .:
Error: R] [MANIFEST] Domain does not match dir name
```

I want to make [OK Example](https://hacs.xyz/docs/publish/integration/#ok-example) but I cant make 2 repos on same dir, which my HASSconfig repo already there

**Comment by ludeeus** [comment]
hassfest does not support the root variant, I invoke hassfest different in this repo to have that working as well here.
So you fix the JSON syntax error it should be ok now

**Comment by xilense** [comment]
ah, thankyou for reply

its make me confuse but I find a lot fun exploring codes

thanks for your hard work!

File: integration [hunk 1]
```diff
@@ -541,6 +541,7 @@
   "willholdoway/hifiberry",
   "wimb0/home-assistant-saj-modbus",
   "WolfRevo/climate.spzb0001_thermostat",
+  "xilense/aimp_custom_component"
```
Comment (ludeeus) [inline integration:544]: This is still not valid JSON

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (xilense) [reply integration]: yeah sorry, my bad missing comma on JSON

I just updated the JSON

**Pull request merged**